tests/test_core.py

```diff
@@ -1273,8 +1273,6 @@ def test_exceptions_allowed(self):
       assert empty_size == fake_size, [empty_size, fake_size]
       # big change when we disable exception catching of the function
       assert size - empty_size > 0.01 * size, [empty_size, size]
-      # full disable can remove a little bit more
-      assert empty_size >= disabled_size, [empty_size, disabled_size]
```

Comment:
I seem to remember having lager change in flight that addresses this too.. (and removes the `if not self.is_wasm():` condition).   But I guess there is no harm in landing this now.